src/main/java/frc/robot/RobotContainer.java

```diff
@@ -577,50 +583,8 @@ public void officialBindings() {
 
     // all need binding
     // Cone
-    driver
-        .a()
-        .onTrue(
-            Commands.sequence(
-                Commands.print("START"),
-                new ClawEncoderMoveDown(-100, clawPivot, clawEncoder, "Cube").withTimeout(1.5),
-                // new PPIDAutoAim(drivetrainSubsystem, limeLightSub, 44),
-                Commands.waitSeconds(.25),
-                Commands.runOnce(mainArm::resetPosition, mainArm),
-                new MagicMotionPos(mainArm, 40, 1, 1, 5),
-                // this one is for cones
-                new MagicMotionPos(mainArm, 190.0, 2.75, 5.0, 1),
-                // for cubes
-                // new MagicMotionPosShuffleboard(mainArm, 210, 2.75, 5, shaftEncoder),
-                Commands.waitSeconds(.25),
-                // new MagicMotionPosShuffleboard(mainArm, 180, 1, 1),
-                // Commands.waitSeconds(),
-                new MagicMotionPos(mainArm, 10, 3, 1.5, .5),
-                new ClawEncoderMoveUp(0, clawPivot, clawEncoder, "Cube"),
-                // Commands.waitSeconds(.3),
-                new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5),
-                new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5)));
-
-    driver
-        .b()
-        .onTrue(
-            Commands.sequence(
-                Commands.print("START"),
-                new ClawEncoderMoveDown(-100, clawPivot, clawEncoder, "Cube").withTimeout(0.5),
-                // new PPIDAutoAim(drivetrainSubsystem, limeLightSub, 44),
-                Commands.runOnce(mainArm::resetPosition, mainArm),
 
-                // this one is for cones
-                new MagicMotionPos(mainArm, 100, 10, 10, 1),
-                // for cubes
-                // new MagicMotionPosShuffleboard(mainArm, 210, 2.75, 5, shaftEncoder),
-                Commands.waitSeconds(.25),
-                // new MagicMotionPosShuffleboard(mainArm, 180, 1, 1),
-                // Commands.waitSeconds(),
-                new MagicMotionPos(mainArm, 10, 3, 1.5, .5),
-                new ClawEncoderMoveUp(0, clawPivot, clawEncoder, "Cube"),
-                // Commands.waitSeconds(.3),
-                new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5),
-                new MagicMotionAbsoluteZero(mainArm, shaftEncoder, 5, 2.5)));
+    driver.b().onTrue(ArmSequenceCommand.starting());
```

Comment:
this should start with lower case - `armSequenceCommand` so we can tell it is not a classname